sdk/lib/crt/math/arm/__64tof_worker.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Implementation of __i64tod
```

Comment:
Better description suggestion, on par with https://github.com/reactos/reactos/blob/master/sdk/lib/crt/math/arm/__fto64.h#L4
```suggestion
 * PURPOSE:     Integer to float conversion (__i64tod/u64tod/i64tos/u64tos)
```